### DISCUSSION TITLE
Docs update: Docs say "CSS Modules are an optional feature", but they don't seem to be

### DISCUSSION THREAD

**Discussion by websocket98765** [vercel/next.js#18075] [Help]
The [NextJS docs](https://nextjs.org/docs/basic-features/built-in-css-support) say: "CSS Modules are an optional feature and are only enabled for files with the `.module.css` extension. Regular <link> stylesheets and global CSS files are still supported." 

But trying to use non-module CSS at the component level gives an error (Next 9.5.5): "Global CSS cannot be imported from files other than your Custom <App>. Please move all global CSS imports to src/pages/_app.js. Or convert the import to Component-Level CSS (CSS Modules). Read more: https://err.sh/next.js/css-global"

The docs are misleading IMO. Modules _are_, unfortunately, required for CSS used at the component level. Needing to move all CSS out of co-located files into a global CSS file doesn't make CSS Modules optional, it means they're required...unless wanting to be really dirty with component-level CSS all dumped into a global file.

**TLDR; I think the [docs](https://nextjs.org/docs/basic-features/built-in-css-support) should ideally be updated, as long as this remains the case, to say: "CSS Modules are required at the component level."**

**Comment by ljosberinn** [comment]
You can use [`<style jsx global>`](https://nextjs.org/blog/styling-next-with-styled-jsx#adding-global-styles) on a per compoennt basis if you really really want to. Global CSS is commonly frowned upon in JS frameworks, so to call this situation "unfortunate" is a bit much and not common consensus.

CSS included in `_app` will be available _across all pages_, which might not be what you want and hurts page loading speed because you carry around CSS that you don't even use on this specific page.

**Reply by websocket98765** [reply]
I'm looking to use `import './Foo.scss';` on the component level b/c I'm converting from CRA to NextJS.

(It is unfortunate IMO because if this were possible it would allow for easier conversion from CRA to NextJS and then to incrementally adopt Next's best practices like CSS modules. I'll probably end up moving all style imports that can't be easily converted to a CSS module into the global css, until it can be rewritten with Tailwind.)

Thanks for the suggestions though.
